acceptance-tests/src/test/java/uk/gov/di/test/step_definitions/Login.java

```diff
@@ -304,4 +304,38 @@ public void theUserEntersTheSixDigitSecurityCodeFromTheirEmail() {
         }
         findAndClickContinue();
     }
+
+    @When("the user enters their {string} code")
+    @When("the new user enters their {string} code")
+    public void theUserEntersTheirAuthCode(String codeType){
+
+        if (codeType == "email") {
+            loginPage.enterSixDigitSecurityCode(System.getenv().get("TEST_USER_EMAIL_CODE"));
```

Comment:
are there any steps (outside of this particular test) that use this step and don't use TEST_USER_EMAIL_CODE and/or TEST_USER_PHONE_CODE

Reply:
Looks like you added the steps and they are only used in this test.